src/sentry/migrations/0067_remove_tracked_superusers.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-04-20 20:03
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+def remove_tracked_superuser_views(apps, schema_editor):
+    """
+    We recently added code to only track alert views of people in the org + member of associated alert projects.
+    This migration removes all of the views we've tracked before adding this change (i.e. superuser views of orgs we're not a part of).
+    """
+    IncidentSeen = apps.get_model("sentry", "IncidentSeen")
+    IncidentProject = apps.get_model("sentry", "IncidentProject")
+    tracked_views = IncidentSeen.objects.all()
```

Comment:
How many records would this pull up? Do you have a risk of running out of memory?

Reply:
I'm thinking no. There are 384 records in this case...but for the future, I'll try to keep this in mind. What kind of memory limits do we actually have? How many records do you think is a reasonable amount to hold in memory?

Reply:
Typically we'd use `RangeQuerySetWrapperWithProgressBar` to limit the number of items in memory. The default pagination size for that is 1000, so probably once we get above that.